sorc/ncep_post.fd/CALLCL.f

```diff
@@ -88,6 +89,7 @@ SUBROUTINE CALLCL(P1D,T1D,Q1D,PLCL,ZLCL)
       DO 30 J=JSTA_M,JEND_M
       DO 30 I=2,IM-1
 !     DO 30 I=1,IM
+      IF(P1D(I,J)<spval.and.Q1D(I,J)<spval.and.Q1D(I,J)<spval)THEN
```

Comment:
you have two "Q1D" here :-)

Reply:
@HuiyaChuang-NOAA Thanks for catching this. Will update.